src/style.css

```diff
@@ -1,3 +1,76 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
```

Comment:
Ensure that the path to the font file is correct and that the font file is available in the specified location. Otherwise, the font may not load properly.